modules/pxp-module-puppet

```diff
@@ -187,53 +87,37 @@ def wait_for_lockfile(lockfile = '', check_interval = 0.1, give_up_after = 30)
   end
 end
 
-def start_run(config, action_input)
+def start_run(action_input)
   exitcode = DEFAULT_EXITCODE
-  cmd = make_command_array(config, action_input)
-  env = make_environment_hash()
-
-  last_run_report = check_config_print("lastrunreport", config)
-  if last_run_report.empty?
-    return make_error_result(exitcode, Errors::NoLastRunReport,
-                             "could not find the location of the last run report")
-  end
+  cmd = Puppet::Util::CommandLine.new('agent', action_input['flags'])
+  Puppet.initialize_settings(cmd.args)
```

Comment:
This also doesn't configure terminuses, which means that no report will be submitted to the master, and that the cached catalog won't be updated.